calcom/cal.com
# feat: Verify email of people setting a meeting with you

## What does this PR do?

Allows event owner to make booker verify their email before scheduling event.

Fixes #9901 

 Loom Video: https://www.loom.com/share/3e9f816ffb6148b09de906080c9fe2fb

## Requirement/Documentation



- Added "requiresBookerEmailVerification" in database.

## Type of change




## How should this be tested?



- Toggle on the Requires Email Verification in advanced settings for an event.
- Now when the booker tries to book the even they are being sent an code to verify.

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
